dali/base/dautils.hpp

```diff
@@ -140,6 +140,7 @@ class da_decl CDfsLogicalFileName
     bool isExpanded() const;
     void expand(IUserDescriptor *user);
     void normalizeName(const char * name, StringAttr &res, bool strict);
+    bool normalizeExternal(const char * name, bool strict);
```

Comment:
In fact, I think you should make both normalizeName and normalizeExternal private now.
Otherwise it's unclear what normalizeName (which expects a non-external) does when passed an external name.

normalizeName() is not called by anything else (only by CDfsLogicalFileName).